python/src/aiconfig/editor/client/src/components/prompt/AddPromptButton.tsx

```diff
@@ -41,30 +41,14 @@ function ModelMenuItems({
 
 export default memo(function AddPromptButton({ addPrompt, getModels }: Props) {
   const [modelSearch, setModelSearch] = useState("");
```

Comment:
Sorry what exactly is `modelSearch` string supposed to represent?

Reply:
There's a search input and `modelSearch` is the text of that input